lib/config.js

```diff
@@ -13,8 +13,9 @@ function Config (process) {
   this.httpProxy = process.env.IED_HTTP_PROXY || process.env.HTTP_PROXY || null
   this.httpsProxy = process.env.IED_HTTPS_PROXY || process.env.HTTPS_PROXY || null
   this.requestRetries = parseInt(process.env.IED_REQUEST_RETRIES, 10) || 5
-  this.sh = process.env.IED_SH || (process.platform === 'win32' ? process.env.comspec || 'cmd' : 'sh')
-  this.shFlag = process.env.IED_SH_FLAG || (process.platform === 'win32' ? '/d /s /c' : '-c')
+  this.sh = process.env.IED_SH || (isWindows ? process.env.comspec || 'cmd' : 'sh')
+  this.shFlag = process.env.IED_SH_FLAG || (isWindows ? '/d /s /c' : '-c')
```

Comment:
It looks like we can skip this entire check: https://nodejs.org/api/fs.html#fs_fs_symlink_target_path_type_callback. It is only available for windows users and should be ignored on other platforms. [Npm](https://github.com/npm/npm/blob/9e03561547ec5830c5f61caa4e6221d4d3b813aa/lib/utils/link.js#L69) is hardcoding it too. Not that we have to follow Npm :smiley: